projects/common/src/navigation/navigation.service.ts

```diff
@@ -229,6 +231,26 @@ export class NavigationService {
     return this.findRouteConfig(path, childRoutes ? childRoutes : []);
   }
 
+  public decorateNavItem(navItem: NavItemConfig, activatedRoute: ActivatedRoute): NavItemConfig {
```

Comment:
can we add some tests for these two methods since they are shared? Can do it in a follow up PR?

Reply:
done.